strings_lab.js

```diff
@@ -0,0 +1,178 @@
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+let variableAmount = 1000000;
+
+if (variableAmount === 1000000){
+  console.log( variableAmount + " dollars " + "(pinky)")
+} else {
+  console.log(variableAmount + " dollars")
+}
+
+// 6. Create a `verbing` code block. It should take a single string variable.
+//If its length is at least 3, it should add 'ing' to its end, unless it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+// ```
+let verbing ="jump";
+if (verbing.endsWith("ing")){
+        let changeWord = verbing.replace("ing","ly")
+        console.log(changeWord)
+} else if (verbing.length >= 3){
+          console.log(verbing + "ing")
+}
+
+
+
+// 7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+
+let string = "itsy bitsy spider";
+
+let newString = string.slice(0,10)
+console.log(newString)
+
+// ## Bonus Questions
+//
+// 1. Write a code block that adds a "#" to a string if the string has an odd number of characters, otherwise leave it as is.
+// print/log the result
+//
+//     Examples:
+//
+//   ```js
+//   let name = 'taq'  // should log 'taq#'
+//   let name2 ='john' // should log 'john'
+
+let strName = "olive";
+
+if (strName.length % 2) {
+  console.log(strName + "#")
+} else {
+  console.log(strName)
+}
+
+
+
+// 2. Write a code block that prints/logs the 3rd to last characters of a string.
+// If the string is not at least 3 characters long then log an error message.
+// <details>
+//     <summary>
+//         Hint
+//     </summary>
+//     It should work for strings of any length.
+// </details>
+//
+// Examples, for variables:
+// ```js
+// let name1 = 'Taq' // Should log 'Taq'
+// let sentence = 'He is Taq'// Should log 'Taq'
+let sentence= "what do you say to the god of death ";
+let name1 ="go"
+let findVar = sentence.includes(name1)
+
+if ( name1.length < 3){
+    console.log("Invalid character amount, enter word with more than 3 characters")
+} else if (findVar === true){
+    console.log(name1)
+} else {
+    console.log("That word can't be found")
+}
+// 3. Write a code block that prints/logs whether a string `x` starts with a vowel.
+// <details>
+//   <summary>
+//     Extra
+//   </summary>
+//
+//   Make it [case insensitive](https://en.wikipedia.org/wiki/Case_sensitivity)
+//
+// </details>
+let begLetter = "apple";
+let gut = begLetter[0];
+    //console.log(gut)
+ if (gut === ( "a" ) || "e"|| "A" || "E"){
```

Comment:
Not a right way to write logical operator. It should be (gut === "a" || gut === "e" || gut === "a" || etc). Also if you need to use multiple logical operators, consider using switch statement.